integration/src/main/java/scripts/testscripts/FolderLifecycle.java

```diff
@@ -154,27 +195,60 @@ protected void doUserJourney(TestUserSpecification testUsers, WorkspaceApi works
     assertEquals(newDescription, updatedFolder2.getDescription());
 
     // Get folder bar.
-    Folder retrievedFolder2 = folderApi.getFolder(getWorkspaceId(), folderBar.getId());
+    Folder retrievedFolder2 = folderOwnerApi.getFolder(getWorkspaceId(), folderBar.getId());
     assertEquals(updatedFolder2, retrievedFolder2);
 
-    // Delete folder bar.
-    folderApi.deleteFolder(getWorkspaceId(), folderBar.getId());
+    // Second user tries to delete folder Loo
```

Comment:
// Second user unable to delete folder Loo. Loo has a private notebook owned by first user. Second user is only workspace writer.